packer/install-dependencies.sh

```diff
@@ -62,8 +62,8 @@ installGPUDrivers
 echo "  - nvidia-docker2 nvidia-container-runtime" >> ${VHD_LOGS_FILEPATH}
 
 VNET_CNI_VERSIONS="
+1.0.30
 1.0.29
-1.0.28
```

Comment:
Let's keep 1.0.28 in the VHD as well to allow folks to choose that version, and for easy emergency rollback w/ VHD backing

Reply:
added it back